sdk/storage/storage-file-share/swagger/README.md

```diff
@@ -846,13 +846,4 @@ directive:
       $["modelAsString"] = false;
 ```
 
```

Comment:
The original PR added another transform here, not sure if it is needed?

```
directive:
  - from: swagger-document
    where: $["parameters"]["AccessTierOptional"]["x-ms-enum"]
    transform: >
      $["modelAsString"] = false;
```

Reply:
Hi @xirzec , I didn't see change here.  I may need more details to identify the changes here...

Reply:
It is likely a mis-titled transformation for xml workaround which is no longer needed

https://github.com/Azure/azure-sdk-for-js/pull/24759/files#diff-00f8d38b77ddcab3d9861608a4e526676d886188f0cd0ff5c746fd187d7a50d3

![image](https://github.com/Azure/azure-sdk-for-js/assets/7583839/0b9e902a-7739-4583-b314-aded649be590)

Reply:
I see the issue here now.

The StringEncoded is used to work around an issue that the old XML parser cannot get the value and attributes back correctly.  I have test cases for it. If all test cases passed, it should be OK to delete this transformation.

Reply:
Right, the bug has been fixed in core